pages/common/docker.md

```diff
@@ -18,3 +18,11 @@
 - Stop a container
 
 `docker stop {{container}}`
+
+- List all images
+
+`docker images`
+
+- Get a shell inside of an image
```

Comment:
Not correct, this would execute the bash command within a new container. The more likely useful command is to get a shell within an already running container, `docker exec -it <name> /bin/bash`

Unfortunately this commands' semantics are quite similar.